js/components/graph/Container.js

```diff
@@ -8,12 +8,12 @@ import PropTypes from "prop-types";
 export default class Container extends React.Component {
   constructor(props) {
     super(props);
-    this.graphRef = React.createRef;
+    this.graph = React.createRef;
   }
   render() {
     return (
       <React.Fragment>
-        <Graph start_blank={this.props.start_blank} edit={this.props.edit} initialDrawMode="draw-node" ref={instance => {this.graphRef = instance; }}/>
+        <Graph start_blank={this.props.start_blank} edit={this.props.edit} initialDrawMode="draw-node" ref={this.graph}/>
         <Sidebar reset={() => this.graph.reset()} />
```

Comment:
can you try doing `this.graph.current.reset()`?

Reply:
Still no dice!